src/styles/styles.css

```diff
@@ -0,0 +1,178 @@
+html {
+  font-family: Roboto, Arial, 'Helvetica Neue', 'Segoe UI', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+}
+
+.card__content {
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  margin-left: 19px;
+  margin-top: 32px;
```

Comment:
The `margin-left` and `margin-top` for `.card__image` can be replaced with more flexible solutions like flexbox alignment or padding on the parent element.